idea_town/frontend/static-src/app/main.js

```diff
@@ -7,12 +8,41 @@ import Me from './models/me';
 import PageManager from './lib/page-manager';
 import Router from './lib/router';
 
+function getDocHeadLink(relName, attrName) {
+  return document
+    .querySelector('head link[rel="' + relName + '"]')
+    .getAttribute(attrName);
+}
+
 app.extend({
+
   initialize() {
+
     app.experiments = new ExperimentsCollection();
     app.experiments.fetch();
```

Comment:
Mind moving this into `dataReady`? If the user doesn't have a session, we don't need to fetch the list of experiments.